src/index.html

```diff
@@ -5,8 +5,428 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="#home">
+          <img src="images/logo.png" alt="header__logo">
+        </a>
+      </div>
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a href="#apple" class="is-active header__link">Apple</a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#samsung" class="header__link">Samsung</a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#smartphones" class="header__link">Smartphones</a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#Lap&com"
+              data-qa="nav-hover"
+              class="header__link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#gadgets" class="header__link">Gadgets</a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#tablets" class="header__link">Tablets</a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#photo" class="header__link">Photo</a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#video" class="header__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="card" data-qa="card">
+        <img src="images/imac.jpeg"
+         alt="imac"
+         class="card__imac"
+        >
```

Comment:
don't forget to start each attribute from new line
```suggestion
        <img
          src="images/imac.jpeg"
          alt="imac"
          class="card__imac"
        >
```